windows/QMK Toolbox/USB.cs

```diff
@@ -152,7 +153,10 @@ public bool DetectBootloader(ManagementBaseObject instance, bool connected = tru
 
             var connectedString = connected ? "connected" : "disconnected";
             var comPortString = comPort != null ? $" [{comPort}]" : "";
-            _printer.Print($"{deviceName} device {connectedString}: {instance.GetPropertyValue("Manufacturer")} {instance.GetPropertyValue("Name")} ({vid}:{pid}:{rev}){comPortString}", MessageType.Bootloader);
+            var driverString = driverName != null ? driverName : "NO DRIVER";
```

Comment:
```suggestion
            var driverString = driverName ?? "NO DRIVER";
```

Should just be able to use null-coalescing here.

Reply:
I always forget that's a thing.